packages/aws-cdk/test/api/hotswap/s3-bucket-hotswap-deployments.test.ts

```diff
@@ -0,0 +1,692 @@
+import { Lambda } from 'aws-sdk';
+import { REQUIRED_BY_CFN } from '../../../lib/api/hotswap/s3-bucket-deployments';
+import * as setup from './hotswap-test-setup';
+
+let mockLambdaInvoke: (params: Lambda.Types.InvocationRequest) => Lambda.Types.InvocationResponse;
+let hotswapMockSdkProvider: setup.HotswapMockSdkProvider;
+
+const payloadWithoutCustomResProps = {
+  RequestType: 'Update',
+  ResponseURL: REQUIRED_BY_CFN,
+  PhysicalResourceId: REQUIRED_BY_CFN,
+  StackId: REQUIRED_BY_CFN,
+  RequestId: REQUIRED_BY_CFN,
+  LogicalResourceId: REQUIRED_BY_CFN,
+};
+
+beforeEach(() => {
+  hotswapMockSdkProvider = setup.setupHotswapTests();
+  mockLambdaInvoke = jest.fn();
+  hotswapMockSdkProvider.setInvokeLambdaMock(mockLambdaInvoke);
+});
+
+test('calls the lambdaInvoke() API when it receives only an asset difference in an s3 bucket deployment', async () => {
```

Comment:
Capitalize things correctly in test descriptions:

```suggestion
test('calls the lambdaInvoke() API when it receives only an asset difference in an S3 bucket deployment', async () => {
```